src/pages/workspace/accounting/netsuite/export/NetSuiteDateSelectPage.tsx

```diff
@@ -0,0 +1,74 @@
+import React, {useCallback, useMemo} from 'react';
+import {View} from 'react-native';
+import type {ValueOf} from 'type-fest';
+import RadioListItem from '@components/SelectionList/RadioListItem';
+import type {ListItem} from '@components/SelectionList/types';
+import SelectionScreen from '@components/SelectionScreen';
+import type {SelectorType} from '@components/SelectionScreen';
+import Text from '@components/Text';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import * as Connections from '@libs/actions/connections/NetSuiteCommands';
+import Navigation from '@navigation/Navigation';
+import type {WithPolicyConnectionsProps} from '@pages/workspace/withPolicyConnections';
+import withPolicyConnections from '@pages/workspace/withPolicyConnections';
+import CONST from '@src/CONST';
+import ROUTES from '@src/ROUTES';
+
+type MenuListItem = ListItem & {
+    value: ValueOf<typeof CONST.NETSUITE_EXPORT_DATE>;
+};
+
+function NetSuiteDateSelectPage({policy}: WithPolicyConnectionsProps) {
+    const {translate} = useLocalize();
+    const policyID = policy?.id ?? '-1';
+    const styles = useThemeStyles();
+    const config = policy?.connections?.netsuite.options.config;
+    const data: MenuListItem[] = Object.values(CONST.NETSUITE_EXPORT_DATE).map((dateType) => ({
+        value: dateType,
+        text: translate(`workspace.netsuite.exportDate.values.${dateType}.label`),
+        alternateText: translate(`workspace.netsuite.exportDate.values.${dateType}.description`),
+        keyForList: dateType,
+        isSelected: config?.exportDate === dateType,
+    }));
+
+    const headerContent = useMemo(
+        () => (
+            <View>
+                <Text style={[styles.ph5, styles.pb5]}>{translate('workspace.netsuite.exportDate.description')}</Text>
+            </View>
+        ),
+        [translate, styles.pb5, styles.ph5],
+    );
+
+    const selectExportDate = useCallback(
+        (row: MenuListItem) => {
+            if (row.value !== config?.exportDate) {
+                Connections.updateNetSuiteExportDate(policyID, row.value, config?.exportDate);
+            }
+            Navigation.goBack(ROUTES.POLICY_ACCOUNTING_NETSUITE_EXPORT.getRoute(policyID));
+        },
+        [config?.exportDate, policyID],
+    );
+
+    return (
+        <SelectionScreen
+            displayName={NetSuiteDateSelectPage.displayName}
+            title="workspace.netsuite.exportDate.label"
+            headerContent={headerContent}
+            sections={[{data}]}
+            listItem={RadioListItem}
+            onSelectRow={(selection: SelectorType) => selectExportDate(selection as MenuListItem)}
```

Comment:
Is there any issue with this?

```suggestion
            onSelectRow={selectExportDate}
```